llvm/lib/Target/DirectX/DXILRootSignature.cpp

```diff
@@ -434,8 +606,28 @@ PreservedAnalyses RootSignatureAnalysisPrinter::run(Module &M,
           OS << indent(Space + 2) << "Flags: " << Descriptor.Flags << "\n";
         break;
       }
+      case llvm::to_underlying(dxbc::RootParameterType::DescriptorTable): {
+        const mcdxbc::DescriptorTable &Table =
+            RS.ParametersContainer.getDescriptorTable(Loc);
+        OS << indent(Space + 2) << "NumRanges: " << Table.Ranges.size() << "\n";
```

Comment:
nit: Suggest avoiding 'magic numbers' and instead use named constants. I assume the 2 is intended to be a tab? Could use 'const uint8_t TabInSpaces = 1'.